samples/client/petstore/cpp-qt5/.clang-format

```diff
@@ -0,0 +1,7 @@
+---
+Language:        Cpp
+AccessModifierOffset: -4
+AllowShortFunctionsOnASingleLine: Empty
+BreakBeforeBinaryOperators: NonAssignment
+ColumnLimit: 0
```

Comment:
The main issue I was dealing with this setting is that if let to default, the description was splitted into several lines which is not easy to do in the mustache template.

Reply:
@MartinDelille Are we going to restore .clang-foramt to the default?

Reply:
I could remove this file since it is not really used.

I used it to help me to generate a better output but was unable to respect the specific `ColumnLimit` rule, in particular when writing the header documentation.

The current template generates:

```
/**
 * OpenAPI Petstore
 * This is a sample server Petstore server. For this sample, you can use the api key `special-key` to test the authorization filters.
 *
 * The version of the OpenAPI document: 1.0.0
 *
 * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
 * https://openapi-generator.tech
 * Do not edit the class manually.
 */
```

Setting `ColumnLimit` to the default would require the following generation:

```
/**
 * OpenAPI Petstore
 * This is a sample server Petstore server. For this sample, you can use the api
 * key `special-key` to test the authorization filters.
 *
 * The version of the OpenAPI document: 1.0.0
 *
 * NOTE: This class is auto generated by OpenAPI Generator
 * (https://openapi-generator.tech). https://openapi-generator.tech Do not edit
 * the class manually.
 */
```

I have no idea how to split the description from the mustache template.

Reply:
> I have no idea how to split the description from the mustache template.

My take is to leverage code formatter to take care of the line split.

Reply:
Yes I got it! This `.clang-format` will only be used if we want to check the template generate a clang format compatible code.